test-testng/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.easymock</groupId>
+    <artifactId>easymock-parent</artifactId>
+    <version>3.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>easymock-test-testng</artifactId>
+  <name>EasyMock Test TestNG</name>
+
+  <properties>
+    <!-- No code coverage need on this project since it's only a test project -->
+    <jacoco.skip>true</jacoco.skip>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.18.1</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.apache.maven.surefire</groupId>
+            <artifactId>surefire-testng</artifactId>
+            <version>2.18.1</version>
+          </dependency>
+        </dependencies>
+        <configuration>
+          <!-- For an unknown reason threadCount is 0 by default and fails with TestNG -->
+          <threadCount>1</threadCount>
```

Comment:
@Tibor17 `threadCount=0` because `threadCount` from `AbstractSurefireMojo` is 0.
From the maven debug, `<threadCount>${threadCount}</threadCount>` and supposed to be empty. I didn't find anythings in pom files of the project that can explain the issue, do you?

I join the debug trace from maven if you want to have a look: [run.txt](https://github.com/easymock/easymock/files/560810/run.txt)